src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -171,6 +171,9 @@ internal static class IDEDiagnosticIds
 
         public const string RemoveUnnecessaryLambdaExpressionDiagnosticId = "IDE0200";
 
+        public const string UseTopLevelStatementsId = "IDE0210";
```

Comment:
Unused?

Reply:
?

Reply:
It **was** unused before 3ea54d0. The comment doesn't make sense starting from that commit